src/src/tx_main.cpp

```diff
@@ -313,8 +313,20 @@ void ICACHE_RAM_ATTR GenerateSyncPacketData()
     --syncSpamCounter;
 }
 
+uint8_t adjustPacketRateForBaud(uint8_t rate)
+{
+  #if defined(Regulatory_Domain_ISM_2400)
+    // Packet rate limited to 250Hz if we are on 115k baud
+    if (crsf.GetCurrentBaudRate() == 115200 && rate == 0) {
```

Comment:
Would this be better as rate > RATE_150HZ?, in case we add faster rates in the future and 500hz is no longer index 0?

Reply:
It would have been nice if the higher the enum value the higher the rate.
But, given thats not the case what about this
```c++
    if (crsf.GetCurrentBaudRate() == 115200) {
      while(rate < RATE_MAX) {
        expresslrs_mod_settings_s *const ModParams = get_elrs_airRateConfig(rate);
        if (ModParams->enum_rate >= RATE_250HZ) {
          break;
        }
        rate++;
      }
    }
```

Reply:
I think that's good